_data/harnesses/anomaly-detection/ksql.yml

```diff
@@ -0,0 +1,125 @@
+dev:
```

Comment:
Would it be appropriate to leverage the new "Short Answer" capabilities, a la https://github.com/confluentinc/kafka-tutorials/blob/master/_data/harnesses/aggregating-count/ksql.yml#L1-L7

Reply:
🤦 I can't believe I forgot this after talking about it last week. I added an answer section.